<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProjectDialog</class>
 <widget class="QDialog" name="AddProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add a new project</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QDateEdit" name="projectDueDE">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="projectStatusCB">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Not Started</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In-Progress</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="projectStartDE">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="projectNameLE"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Project Status: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="projectDescTE"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Project Due Date: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Project Start Date: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Project Description: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Project Name: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="addProjectStatusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addProjectButton">
          <property name="text">
           <string>Add Project</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitProjectButton">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
